raiden_libs/network/matrix/client.py

```diff
@@ -21,31 +23,60 @@ class GMatrixHttpApi(MatrixHttpApi):
     A wrapper around MatrixHttpApi to limit the number
     of concurrent requests we make to the number of connections
     available to us in requests.Session connection pool size.
+
+    Args:
+        pool_maxsize: max size of underlying/session connection pool
+        retry_timeout: for how long should a single request be retried if it errors
+        retry_delay: delay to way between a retries inside a request.
```

Comment:
That sentence is not English ;)

Reply:
Haha, some cut-pase didn't go as expected